e2e/cypress/tests/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -805,41 +805,6 @@ describe('Messaging', () => {
             cy.get('#edit_textbox', {timeout: TIMEOUTS.FIVE_SEC}).type('{enter}');
         });
     });
-
-    it('MM-T2227 Channel shortlinking - can edit', () => {
```

Comment:
Was this feature not replaced/moved?

Reply:
The link has been completely removed from the RHS